alloc.h

```diff
@@ -6,4 +6,19 @@
 #define alloc(x) malloc(x)
 #define alloc_free(x) free(x)
 
+/*
+ * This functions should not be used in any new code. It is provided so that
+ * external patches that rely on it's presence still work correctly.
```

Comment:
How about: “In-tree code must not call this deprecated function. It’s here because some external patches still do. We intend to remove it as soon as is practical.”

Reply:
Done.